.github/workflows/upload-wheels.yml

```diff
@@ -0,0 +1,81 @@
+name: Build
+
+on:
+  push:
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+'
+
+
+env:
+  CIBW_MANYLINUX_X86_64_IMAGE: manylinux2014
+  CIBW_BUILD_VERBOSITY: 2
+  # Only build on Python 3.x
+  CIBW_BUILD: 'cp3?-*'
+  CIBW_SKIP: 'cp35-* cp39-* *-manylinux_i686'
+  CIBW_BEFORE_TEST: pip install -r tests/requirement_tests.txt
+  CIBW_TEST_COMMAND: nosetests {project}/tests
+  MACOSX_DEPLOYMENT_TARGET: '10.13'
+
+jobs:
+  build_wheels:
+    name: Build wheels on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-18.04, macos-latest]
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: 'true'
+
+      - uses: actions/setup-python@v2
+        name: Install Python
+        with:
+          python-version: '3.7'
+
+      - name: Install cibuildwheel
+        run: |
+          python -m pip install cibuildwheel==1.6.4
+
+      - name: Build wheels on Linux
+        if: runner.os == 'Linux'
+        env:
+          CIBW_BEFORE_BUILD: 'yum update -y; yum -y install hdf5-devel'
```

Comment:
can you check the produced wheel to make sure it's using a static hdf5 version (and not a rpath versioned .so); otherwise we may run into problems with conflicts of people using different hdf5 versions.